jobs/integration/conftest.py

```diff
@@ -232,7 +232,7 @@ async def k8s_model(model, tools):
             "--no-switch",
         )
         created_k8s_model = True
-        k8s_model = Model(model.loop)
+        k8s_model = Model()
```

Comment:
This exposed the same problem.  Starting in juju 2.9.5, Model's first argument was no longer an event_loop, but instead `max_frame_size`